ethereum/core/src/main/java/org/hyperledger/besu/ethereum/bonsai/BonsaiWorldStateUpdater.java

```diff
@@ -714,7 +725,8 @@ public boolean isDirty() {
         && deletedAccounts.isEmpty()
         && storageToUpdate.isEmpty()
         && storageToClear.isEmpty()
-        && codeToUpdate.isEmpty());
+        && codeToUpdate.isEmpty()
+        && emptySlot.isEmpty());
```

Comment:
we don't need this since we do not persist anything about emptySlot collection.  Even if it isn't empty, it does not make this updater dirty.  If it isn't empty, but nothing else has changed we might unnecessarily trigger a roothash calc.

Reply:
Good point